packages/account-v2/src/components/FormFields/__tests__/FormInputField.spec.tsx

```diff
@@ -0,0 +1,46 @@
+import React from 'react';
+import { Form, Formik } from 'formik';
+import { render, screen } from '@testing-library/react';
+import userEvent from '@testing-library/user-event';
+import FormInputField from '../FormInputField';
+
+describe('FormInputField', () => {
+    it('should render the Input field', () => {
+        render(
+            // eslint-disable-next-line @typescript-eslint/no-empty-function
+            <Formik initialValues={{}} onSubmit={() => {}}>
```

Comment:
Can we assign `jest.fn` to onSubmit rather than an empty function?

Reply:
Updated.